git_patrol.py

```diff
@@ -242,20 +249,22 @@ def log_command_error(command, returncode, stdout_bytes, stderr_bytes):
   gcb_log_subproc = await commands.gcloud(
       'builds', 'log', '--stream', '--no-user-output-enabled',
       str(cloud_build_uuid))
-  await gcb_log_subproc.communicate()
+  stdout_bytes, stderr_bytes = await gcb_log_subproc.communicate()
   returncode = await gcb_log_subproc.wait()
   if returncode:
-    logger.warning('gcloud builds log returned %d', returncode)
+    log_command_error(
+        'gcloud builds log', returncode, stdout_bytes, stderr_bytes)
```

Comment:
I think it looks a little cleaner this way rather than performing the decode outside the function. We don't need to decode stderr_bytes for example, except for logging in this function.